src/persist-client/src/fetch.rs

```diff
@@ -1319,7 +1328,12 @@ impl Cursor {
         &self,
         encoded: &'a EncodedPart<T>,
     ) -> Option<(&'a [u8], &'a [u8], T, [u8; 8])> {
-        let part = encoded.part.updates.records();
+        // TODO(bkirwi): replace before allowing structured-only parts
```

Comment:
A pattern I've liked recently is naming the TODOs after the feature, e.g. `TODO(structured)`, makes it really easy to find these things again. Wdyt?